paasta_tools/cli/cmds/docker_inspect.py

```diff
@@ -30,11 +31,15 @@ def add_subparser(subparsers):
 
 
 def paasta_docker_inspect(args):
-    status = get_status_for_instance(cluster=args.cluster,
-                                     service=args.service,
-                                     instance=args.instance)
-    slave = pick_slave_from_status(status=status,
-                                   host=args.host)
-    container = get_container_name(status.marathon.app_id, slave_hostname=slave, task_id=args.mesos_id)
+    try:
+        task = get_task_from_instance(cluster=args.cluster,
+                                      service=args.service,
+                                      instance=args.instance,
+                                      slave_hostname=args.host,
+                                      task_id=args.mesos_id)
+    except TaskNotFound:
+        sys.exit(1)
```

Comment:
there's some logging before the exception is thrown, so not much more to say here.